src/app/app.component.ts

```diff
@@ -124,6 +124,15 @@ export class AppComponent {
             this.resetOgpTags();
           }
 
+          // Show Ethnio survey on asset, search, collection, browse-groups, group routes
```

Comment:
@rehanabbasi I'm not sure we've used this approach before, as opposed to calling load script per component. I like it!